src/chainparams.cpp

```diff
@@ -341,6 +355,12 @@ class CRegTestParams : public CChainParams {
         consensus.nPowMaxAdjustDown = 0; // Turn off adjustment down
         consensus.nPowMaxAdjustUp = 0; // Turn off adjustment up
         consensus.nPowTargetSpacing = 2.5 * 60;
+        consensus.vUpgrades[Consensus::BASE_SPROUT].nActivationHeight =
+            Consensus::NetworkUpgrade::ALWAYS_ACTIVE;
+        consensus.vUpgrades[Consensus::UPGRADE_TESTDUMMY].nActivationHeight =
+            Consensus::NetworkUpgrade::NO_ACTIVATION_HEIGHT;
+        consensus.vUpgrades[Consensus::UPGRADE_OVERWINTER].nActivationHeight =
+            Consensus::NetworkUpgrade::NO_ACTIVATION_HEIGHT;
```

Comment:
At the moment, this means that all the RPC tests will default to Sprout mode. I think we should at some point change this to `ALWAYS_ACTIVE` (updating `src/gtest/test_upgrades.cpp` accordingly), and have a few specific RPC tests that disable Overwinter.